### PR TITLE
Fix linting issues in pkg/kubectl/cmd/create

### PR DESCRIPTION
**Pull request by agowa**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Fix golint errors in `pkg/kubectl/cmd/create`

**Which issue(s) this PR fixes**:

See #68026 for more information.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

Resubmit of #73116

### CODE REVIEW

**Comment by xichengliudui** [comment]
Can you fix these conflicts?

**Comment by hvaara** [comment]
@agowa338 Thanks for the PR!
There seems to be an [issue](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/78229/pull-kubernetes-typecheck/1189429282254884865#1:build-log.txt%3A17) with the test. Can you please take a look?

**Comment by agowa338** [comment]
Sorry for the delay, I was not that active on github lately and I must have missed the notification email.

I fixed the failing linting test. I'll have a look at the other two failing tests this week.

File: pkg/kubelet/apis/podresources/v1alpha1/api.pb.go [hunk 1]
```diff
@@ -17,36 +17,23 @@ limitations under the License.
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: api.proto
 
-/*
```
Comment (dashpole) [inline pkg/kubelet/apis/podresources/v1alpha1/api.pb.go]: Can you confirm that these updates were made by running `./hack/update-generated-pod-resources.sh`  I'm surprised it removed the comments at the top.

**Pull request closed without merging**